src/serializer/ResidualFunctions.js

```diff
@@ -427,7 +427,6 @@ export class ResidualFunctions {
           let firstUsage = this.firstFunctionUsages.get(functionValue);
           invariant(insertionPoint !== undefined);
           if (
-            this.residualFunctionInitializers.hasInitializerStatement(functionValue) ||
```

Comment:
I think this was the only callsite of `hasInitializerStatement`. Please remove the function declaration as well, or even better, add the invariant `invariant(!this.residualFunctionInitializers.hasInitializerStatement(functionValue))` to the else-branch.